staff/ruben-ponce/precourse/4-skylab-airlines-pro-master/index.js

```diff
@@ -0,0 +1,196 @@
+'use strict'
+var flights = [
+  { id: 00, to: 'Bilbao', from: 'Barcelona', cost: 1600, scale: false },
+  { id: 01, to: 'New York', from: 'Barcelona', cost: 700, scale: false },
+  { id: 02, to: 'Los Angeles', from: 'Madrid', cost: 1100, scale: true },
+  { id: 03, to: 'Paris', from: 'Barcelona', cost: 210, scale: false },
+  { id: 04, to: 'Roma', from: 'Barcelona', cost: 150, scale: false },
+  { id: 05, to: 'London', from: 'Madrid', cost: 200, scale: false },
+  { id: 06, to: 'Madrid', from: 'Barcelona', cost: 90, scale: false },
+  { id: 07, to: 'Tokyo', from: 'Madrid', cost: 1500, scale: true },
+  { id: 08, to: 'Shangai', from: 'Barcelona', cost: 800, scale: true },
+  { id: 09, to: 'Sydney', from: 'Barcelona', cost: 150, scale: true },
+  { id: 10, to: 'Tel-Aviv', from: 'Madrid', cost: 150, scale: false }
+];
+
+//Bienvenida
```

Comment:
Hey Ruben, you did a great job naming the function with a clear name, so this comment is not necessary.

Reply:
solved, ty!